### ISSUE TITLE
default keyword missing

### ISSUE THREAD

**Issue by krilbe** [davideicardi/DynamicExpresso#42]
It seems there's currently no support for this:

`default(int)`

It reports unknown identifier 'default'.

Can we have support for `default` keyword added?

**Comment by davideicardi** [comment]
Yes, for now it is not supported. It can be probably added without problems. I will put this on my TODO list for the future. Of course any help is appreciated!

**Comment by JustinRChou** [comment]
I am running a test on my update and I was trying default(float) which was failing as it is not in _knownTypes dict. Double was working.

**Comment by krilbe** [comment]
As of 2.3.1 default(int?) still doesn't work. What's the status of the default keyword in general? Works for some types but not all?

**Comment by davideicardi** [comment]
For now 'default' operator is still not supported. I'm sorry. If someone can help me on this with a PR I will appreciate.

**Comment by krilbe** [comment]
Alright. Not a big deal, I just wanted to know. :-) I worked around it using this:
`(int?)null`
